From ffc2799d1144bae65012529ee5928c8293103fab Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Thu, 14 Apr 2011 09:39:38 -0400
Subject: [PATCH] drivers/net: block failing combo of legacy E1000 and IGB

The upgrade of the IGB driver to intel's latest causes a
large class of function naming conflicts between the IGB
driver and the legacy PCI/PCI-X e1000 driver.

Block the ability to have them both built-in at the same time.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>

diff --git a/drivers/net/Kconfig b/drivers/net/Kconfig
index 7834cf5..fbb3570 100644
--- a/drivers/net/Kconfig
+++ b/drivers/net/Kconfig
@@ -2113,7 +2113,7 @@ config IP1000
 
 config IGB
        tristate "Intel(R) 82575/82576 PCI-Express Gigabit Ethernet support"
-       depends on PCI
+       depends on PCI && !(E1000=y)
        ---help---
          This driver supports Intel(R) 82575/82576 gigabit ethernet family of
          adapters.  For more information on how to identify your adapter, go
-- 
1.7.4.4

